src/javascript/app/pages/trade/purchase.js

```diff
@@ -79,8 +79,8 @@ const Purchase = (() => {
             let formula, multiplier;
             const {contract_type} = passthrough;
             if (isLookback(contract_type)) {
-                multiplier = formatMoney(currency, passthrough.amount);
-                formula    = getLookBackFormula(contract_type, multiplier, false, 3, 2);
+                multiplier = formatMoney(currency, passthrough.amount, multiplier, false, 3, 2);
```

Comment:
Seems `multiplier` shouldn't be in the list of arguments?

Reply:
@binary-hamedanchi yeah.